stable/redmine/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: redmine
-version: 0.4.3
+version: 0.5.3
```

Comment:
@tompizmor @prydonius Did we change the versioning scheme? Shouldn't that be `0.5.0`